### PR TITLE
AC-1020: Add Test For PatientMerger

### PR DESCRIPTION
**Pull request by Rohit-2602**
## Description of what I changed
Add Test case for PatientMerger
Contains 2 test case - 
- Patient Fields not null or "null"
- Some Patient Fields are null or "null"

## Issue I worked on

JIRA Issue: https://issues.openmrs.org/browse/AC-1020

## Checklist: I completed these to help reviewers :)

- [x] My pull request only contains **ONE single commit**
(the number above, next to the 'Commits' tab is 1).

- [x] I have **added tests** to cover my changes. (If you refactored
existing code that was well tested you do not have to add tests)

- [x] All new and existing **tests passed**.

- [x] My pull request is **based on the latest changes** of the master branch.

### CODE REVIEW

**Comment by Rohit-2602** [comment]
@LuGO0 Please review

File: openmrs-client/src/test/java/org/openmrs/mobile/utilities/PatientMergerTest.kt [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+        assertEquals(oldPatient.id, newPatient.id)
+
+        assertNotEquals(oldPatient.name, newPatient.name)
+        assertNotEquals(oldPatient.address, newPatient.address)
+        assertNotEquals(oldPatient.birthdate, newPatient.birthdate)
+        assertNotEquals(oldPatient.gender, newPatient.gender)
+    }
+
+    @Test
+    fun `non null fields remain same and null fields changes after merge`() {
```
Comment (LuGO0) [inline openmrs-client/src/test/java/org/openmrs/mobile/utilities/PatientMergerTest.kt:29]: @Rohit-2602 pick up the testing conventions from the already written tests and then refractor these tests to abide by those conventions.

**Review by LuGO0 — CHANGES_REQUESTED** [review]

**Comment by Rohit-2602** [comment]
@LuGO0 This is the naming convention for Kotlin test cases.
More here - https://kotlinlang.org/docs/coding-conventions.html#names-for-test-methods.
This makes the test case more readable.

**Comment by LuGO0** [comment]
> @LuGO0 This is the naming convention for Kotlin test cases. More here - https://kotlinlang.org/docs/coding-conventions.html#names-for-test-methods. This makes the test case more readable.

Lets use the second option mentioned there dont want to have multiple different conventions in the codebase `ensureEverythingWorks_onAndroid()`

**Pull request merged**